runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -315,6 +315,21 @@ public void handleJavaScriptResult(String jsonResult) {
         mContentViewCore.getWebContents().evaluateJavaScript(script, coreCallback);
     }
 
+    public void evaluateJavascriptForTests(String script, ValueCallback<String> callback) {
+        if (mNativeContent == 0) return;
+        final ValueCallback<String>  fCallback = callback;
+        JavaScriptCallback coreCallback = null;
+        if (fCallback != null) {
+            coreCallback = new JavaScriptCallback() {
+                @Override
+                public void handleJavaScriptResult(String jsonResult) {
+                    fCallback.onReceiveValue(jsonResult);
+                }
+            };
+        }
+        mContentViewCore.getWebContents().evaluateJavaScriptForTests(script, coreCallback);
+    }
```

Comment:
Above added code seems totally dup with evaluateJavascript except two final variables. Can we re-use code somehow?

And please add comment to mention this new API for testing only, and why it resolve the problem.